.bats/entities.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd $HOME/ardrive-cli/
+
+cache="$(yarn ardrive list-all-drives -w $WALLET)"
+driveID="$(echo $cache | jq '[.. | select(.drivePrivacy? == "public")][0] | .driveId' | cut -d '"' -f 2)"
+folderID="$(echo $cache | jq '[.. | select(.drivePrivacy? == "public")][0] | .rootFolderId' | cut -d '"' -f 2)"
```

Comment:
I tested it in isolation and works well. Although I'd like to get rid of that optional field (i.e. `?`) so we enforce the input to be in good shape
```suggestion
driveID="$(echo $cache | jq --raw-input '[.[] | select(.drivePrivacy == "public")][0] | .driveId')"
folderID="$(echo $cache | jq --raw-input '[.[] | select(.drivePrivacy == "public")][0] | .rootFolderId')"
```
**Explanation**
1. removing the question mark throws an error (cannot index an array with a string value)
2. replacing the recursive operator by explicitly iterating the expected array. `..` -> `.[]` 
3. we get rid of the call to `cut` to remove the quotes by using the `-r --raw-input` flag at `jq`